server.config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  PORT: 4001,
+  TESTING_PORT: 8001,
+};
```

Comment:
What do you think about keeping the `port` key consistent, and specifying the environment as the top level key?

```js
module.exports = {
  development: { port: 4001 },
  test: { port: 8001 },
}
```